bindings/rust/Cargo.toml

```
@@ -15,3 +15,8 @@ edition = "2018"
 [build-dependencies]
 bindgen = "0.54.0"
 cmake = "0.1"
+
+[badges]
+circle-ci = { repository = "https://github.com/wasmx/fizzy" }
```

Not sure if this should be:
```suggestion
circle-ci = { repository = "wasmx/fizzy" }
```

---

I don't know either.